travis/cfg/travis_run_pylint_beta.cfg

```diff
@@ -2,16 +2,21 @@
 #  pylint configuration file will display them without affecting your build status.
 
 # Beta message and code:
+#   api-one-deprecated - W8104
```

Comment:
@dreispt 
Can you check [CONTRIBUTING.md#field](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#field)?
`@api.one method: For v8 is recommended use @api.multi and avoid use @api.one, for compatibility with v9 where is deprecated @api.one.`